CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.21.0-rc.0 / 2020-08-26
+
+* [CHANGE] Promtool: Add tsdb commands as promtool tsdb subcommand. #6088
+* [FEATURE] Docker Swarm SD: Support tasks and service without published ports. #7686
+* [FEATURE] Eureka SD: New service discovery. #3369
+* [FEATURE] Hetzner SD: New service discovery. #7822
+* [FEATURE] Kubernetes SD: Support Kubernetes EndpointSlices. #6838
```

Comment:
Is this big enough to be a feature?

Reply:
I think so, and I have checked that with @brancz too.